share_extension/icons/excel.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {View} from 'react-native';
+import Svg, {
+    G,
+    Path
+} from 'react-native-svg';
+
+const ExcelSvg = ({height, width}) => { //eslint-disable-line
+    return (
+        <View style={{height, width, alignItems: 'flex-start'}}>
+            <Svg
+                width={width}
+                height={height}
+                viewBox='0 0 24 32'
+            >
+                <G
+                    stoke='none'
```

Comment:
Aren't these supposed to be `stroke` and `strokeWidth` instead of `stoke` and `stokeWidth`? This happens in all of these files